pkg/test/application/echo/echo.go

```diff
@@ -87,35 +91,46 @@ func (a *echo) start() (err error) {
 		return err
 	}
 
-	a.servers = make([]serverInterface, len(a.ports))
-	for i, p := range a.ports {
-		handler := &handler{
-			version: a.version,
-			caFile:  a.tlsCert,
-			dialer:  a.dialer,
-		}
-		switch p.Protocol {
-		case model.ProtocolTCP:
-			fallthrough
-		case model.ProtocolHTTP:
-			fallthrough
-		case model.ProtocolHTTPS:
-			a.servers[i] = &httpServer{
-				port: p,
-				h:    handler,
-			}
-		case model.ProtocolHTTP2:
-			fallthrough
-		case model.ProtocolGRPC:
-			a.servers[i] = &grpcServer{
-				port:    p,
-				h:       handler,
-				tlsCert: a.tlsCert,
-				tlsCKey: a.tlsCKey,
-			}
-		default:
-			return fmt.Errorf("unsupported protocol: %s", p.Protocol)
-		}
+	a.servers = make([]serverInterface, 0)
+	//for i, p := range a.ports {
+	//	handler := &handler{
+	//		version: a.version,
+	//		caFile:  a.tlsCert,
+	//		dialer:  a.dialer,
+	//	}
+	//	switch p.Protocol {
+	//	case model.ProtocolTCP:
+	//		fallthrough
+	//	case model.ProtocolHTTP:
+	//		fallthrough
+	//	case model.ProtocolHTTPS:
+	//		a.servers[i] = &httpServer{
+	//			port: p,
+	//			h:    handler,
+	//		}
+	//	case model.ProtocolHTTP2:
+	//		fallthrough
+	//	case model.ProtocolGRPC:
+	//		a.servers[i] = &grpcServer{
+	//			port:    p,
+	//			h:       handler,
+	//			tlsCert: a.tlsCert,
+	//			tlsCKey: a.tlsCKey,
+	//		}
+	//	default:
+	//		return fmt.Errorf("unsupported protocol: %s", p.Protocol)
+	//	}
+	//}
```

Comment:
did you mean to leave all this commented-out code?

Reply:
that explains why the tests failed!.. this was supposed to be uncommented